internal/backend/db/dbfactory/factory.go

```diff
@@ -24,8 +24,7 @@ var Configs = configset.Set[Config]{
 		Type:               gormkitteh.RequireExplicitDSNType,
 	},
 	Dev: &Config{
-		Type: "sqlite",
-		DSN:  "file:" + filepath.Join(xdg.DataHomeDir(), "autokitteh.sqlite"),
+		DSN: "sqlite:file:" + filepath.Join(xdg.DataHomeDir(), "autokitteh.sqlite"),
```

Comment:
I remember that somewhere there was an explicit check of the type value. Do you know where? Is that changed?

Reply:
I think it's in gormkitteh, but there is a call for `explicit()` which will parse `sqlite` as type from DSN

Reply:
yeah and he doesn't like it :) wdyt?

Reply:
let's discuss https://linear.app/autokitteh/issue/ENG-769/gormkitteh-db-type-in-dsn